_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+It should look a little something like this:
+{
+
+
+
+  Picture Goes here:
+
+
+
+}
+
+You might ask, what about alerts? I'm going to wager your alerts are being pulled into your OBs via browser source. You set these up your normal way and when you do, make sure to set Control Audio Via OBS. This will give you an audio level bar in the Audio Mixer portion of OBS.
+
+{
+
+
+  picture of above here:
+
+
+
+}
+
+You will notice a cog options menu icon beside the specific Audio Source in the Audio Mixer. Click this and you will see a menu and something that reads Advanced Audio Properties which opens a window like this:
+
+{
+
+
+
+  picture of above Here
+
+
+
+}
+
+Select 'Monitor and Output' for the Alerts you want to hear. Then go into Settings->Audio->Advanced Section->Monitoring Device: Set this to 'VoicMeeter Aux Input (VB-Audio VoiceMeeter Aux VAIO)'
+
+{
+
+
+
+  picture of above here
+
+
+
+}
+
+Now go to your Voicemeeter Banana application and test some Alerts. You should see the Voicemeeter AUX audio bouncing around. Since we are pulling the audio into OBS via Browser Source and both Outputting and Monitoring, all we need is to push this audio to our headphones so we can hear when things happen. So select whichever A# is your headphones underneath Voicemeeter AUX and test again to make sure you ca hear it.
+
+I believe this covers the whole setup for using Voicemeeter Banana and OBS. At a later point I will be going over how to add Discord and other applications to the mix. At some point I may cover a Potato version setup. Speaking of Potato, if you feel like the amount of inputs / ouputs are not enough you can give Potato a shot. The setup is very similar. I covered the differences up above but it does have a small cost. If you don't need to or don't want to invest money into it in the next 30 days or so, you could be just fine with Banana for free.
+
+This article covered the bare bones of Voicemeeter and should be good to get you started. If you want to know about specific setups that can be done, please come ask about them in the discord. At some point I will be adding more specific setups in another post including: Setting Up Discord With Voicemeeter, Setting up Game Mic Audio with Voicemeeter, Setting Specific Output of Apps to Specific Digital Audio Cables.
+
+I am going to guess you already know about audio leveling and adjustments and how to set that up. If not we will be doing an article on that as well. Or if you would like help with it prior to us releasing the article we are available in the discord! yay another discord plug lol
```

Comment:
If people are wondering about Voicemeeter, I'd assume they don't know about audio leveling and the correct "mix" there.

Reply:
I think I also dropped this bit of fluff as well in the new article revisions